src/SFA.DAS.DATA.MANAGEMENT/StoredProcedure/LoadCandidateEthLookup.sql

```diff
@@ -41,7 +41,7 @@ BEGIN TRY
 IF @@TRANCOUNT=0
 BEGIN
 BEGIN TRANSACTION
-DELETE FROM dbo.CandidateEthLookUp
+TRUNCATE TABLE dbo.CandidateEthLookUp
```

Comment:
has this been checked - asking as the permission granted recently was only on AsDataPl schema only?